src/content/contributing/release-process/_index.en.md

```diff
@@ -79,8 +79,8 @@ projects require, you will need to create a pre-release (for example `v0.5.0-pre
 
 2) In the `shipyard` project, edit `package/Dockerfile.shipyard-dapper-base` and set `SUBCTL_VERSION` to the new version.
 
-3) Commit the change and create a pull request with the `test-projects` label so it runs the E2E tests on the consuming
-   projects as well. After the tests successfully complete, have it merged.
+3) Commit the change and create a pull request with the `e2e-projects` and `lint-projects` labels so it runs the E2E tests and all linting
```

Comment:
Do we need to run all linting in this case?

Reply:
I don't know. I guess our assumption is that exactly that code has already been tested, enforced by our PR GH config. Likely only the MD links check could fail. My assumption was that we wanted to do as full a verification here as possible, and that the cost of running linting wouldn't be significant compared to the E2E. It might be good insurance against unknown-unknowns.

Reply:
We bump the SUBCTL_VERSION here and want to ensure downstream E2E doesn't break.  Linting doesn't come into play here wrt downstream projects, unless I'm missing something.

Reply:
Okay, removed the linting parts in the latest.